unfurl/configurators/octodns.py

```diff
@@ -0,0 +1,199 @@
+import logging
+import os
+from collections import MutableMapping
+from contextlib import contextmanager
+from dataclasses import dataclass
+from pathlib import Path
+
+from octodns.manager import Manager
+from ruamel.yaml import YAML
+
+from unfurl.configurator import Configurator
+from unfurl.job import ConfigTask
+from unfurl.projectpaths import WorkFolder
+
+log = logging.getLogger(__file__)
+
+OPERATION = None  # hack for unit tests
```

Comment:
How to define "operation" in unit tests in better way? This hack is ugly. :/

Reply:
you should not need something like that -- take a look at the other tests for configurators, for example test_terraform -- it tests all the operations by running the ensemble through the deployment life cycle... deploy, check, reconfigure and redeploy, undeploy... (btw the intent of issue #52is to create a test driver so we don't have to keep repeating that pattern)